app/views/layouts/application.html.haml

```diff
@@ -12,3 +12,6 @@
         = yield
 
     = render 'shared/footer'
+
+  :javascript
+  var navigation = $('#nav-main').okayNav();
```

Comment:
Also this should not be inline javascript. Move to a javascript file that already exists.